flake.nix

```diff
@@ -57,6 +57,7 @@
             if [ $OPENAI_API_KEY == "" ]; then
               echo "Please go to https://beta.openai.com/account/api-keys then set the OPEN_API_KEY environment variable"
             fi
+            exec $SHELL
```

Comment:
For whatever reason nix forces bash in its dev shell for now [its going to be fixed](https://github.com/NixOS/nix/pull/8043) but not quite yet

Reply:
Seems like a feature to me ;)